src/ports/postgres/modules/lda/lda.py_in

```diff
@@ -208,15 +232,19 @@ class LDATrainer:
                             doc_topic[{topic_num} + 1:array_upper(doc_topic, 1)],
                             {voc_size},
                             {topic_num}
-                        ) AS model
+                        ) AS model,
```

Comment:
1. Adding a comma after `model` makes the query syntactically wrong since n_iterations already has a comma at the end.
1. Adding a comma after `perplexity_iterations` will also make the syntax invalid since there will be a comma right before the FROM clause

Reply:
done.